config/db.js

```diff
@@ -0,0 +1,31 @@
+const mongoose = require('mongoose');
+const express = require('express');
+const dbinitRoute = require('../src/routes/dbinitRoute');
+
+const app = express();
+
+const PORT = 3000;
+const DB_URL = 'mongodb+srv://admin:admin@cluster0.jg02e.mongodb.net/myFirstDatabase?retryWrites=true&w=majority';
+
+app.use(dbinitRoute);
+
+async function runDatabase() {
+  try {
+    await mongoose.connect(
+      DB_URL,
+      {
+        useNewUrlParser: true,
+        useFindAndModify: false,
+      },
+    );
+    app.listen(PORT, () => {
```

Comment:
the creation of server should be done in `index.js`, the variable `PORT` should be there too

this file contains the **database** information only 

 function `runDatabase()` should connect to db only. it mustn't run a server